tslint.json

```diff
@@ -1,31 +1,64 @@
 {
   "rules": {
+    "array-type": [true, "array-simple"],
+    "arrow-return-shorthand": true,
+    "ban": [true,
+      {"name": "parseInt", "message": "tsstyle#type-coercion"},
+      {"name": "parseFloat", "message": "tsstyle#type-coercion"},
+      {"name": "Array", "message": "tsstyle#array-constructor"}
+    ],
+    "ban-types": [true,
+      ["Object", "Use {} instead."],
```

Comment:
What's the rationale for this rule? Just curious.

Reply:
+1, as a reader I prefer `Object`

Reply:
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

Reply:
More: https://stackoverflow.com/questions/18961203/typescript-any-vs-object/28795689#28795689